packages/ui/src/components/AppContent/index.tsx

```diff
@@ -29,9 +29,18 @@ const AppContent = ({ children }: Props) => {
     document.body.classList.add(styles[theme] ?? '');
```

Comment:
`remove` and `add` support multiple parameters. may combine them

Reply:
updated